sklearn/metrics/tests/test_common.py

```diff
@@ -545,6 +553,8 @@ def precision_recall_curve_padded_thresholds(*args, **kwargs):
     "mean_poisson_deviance",
     "mean_gamma_deviance",
     "mean_compound_poisson_deviance",
+    "mean_squared_log_error",
+    "root_mean_squared_log_error",
```

Comment:
Should these be added because I think we can have negative values that are above -1 like -0.5?